arch.mmd

```diff
@@ -0,0 +1,23 @@
+%% Ensure when adding entities here they match what's defined in the internal-docs repo
+
+subgraph Core
+  core --> core_maria_db
+end
+
+subgraph Resolver
+  resolver --> resolver_maria_db
+end
+
+subgraph Device Registry
+  device_registry --> device_registry_maria_db
+end
+
+core --> auth_plus
+core --> device_registry
+core --> kafka
+core --> package_store
+core --> resolver
+core --> tuf_repo
+core --> user_profile
+device_registry --> kafka
+resolver --> core
```

Comment:
I don't think there's a super nice way to say "listens vs. publishes" since we don't differentiate channels, so I simplified in it's always `$service --> kafka`.

And you mean remove `resolver --> core` and add `resolver --> device_registry`?